lib/config/types.ts

```diff
@@ -411,6 +411,10 @@ export type MergeStrategy =
   | 'rebase-merge'
   | 'squash';
 
+// ref: https://github.com/renovatebot/renovate/issues/39458
+// This list should be added to as any new unsafe execution commands should be permitted
+export type AllowedUnsafeExecution = 'goGenerate' | 'gradlewExecution';
```

Comment:
Because we've not yet implemented `go:generate` support:

```suggestion
export type AllowedUnsafeExecution = 'gradlewExecution';
```

Reply:
yup, i just wanted to ensure I was on the right track.